sources/zafira-dbaccess/src/main/java/com/qaprosoft/zafira/dbaccess/dao/mysql/application/search/UserSearchCriteria.java

```diff
@@ -15,51 +15,14 @@
  *******************************************************************************/
 package com.qaprosoft.zafira.dbaccess.dao.mysql.application.search;
 
-import com.qaprosoft.zafira.models.db.User;
-
 import java.util.Date;
 
-public class UserSearchCriteria extends SearchCriteria implements DateSearchCriteria
+public class UserSearchCriteria extends BaseSearchCriteria implements DateSearchCriteria
```

Comment:
Let's not use intermediate BaseSearchCriteria and move all that new stuff to SearchCriteria.

Reply:
@hursevich moved all usages to SearchCriteria